modules/tinymce/src/plugins/link/test/ts/browser/QuickLinkTest.ts

```diff
@@ -156,4 +159,26 @@ describe('browser.tinymce.plugins.link.QuickLinkTest', () => {
       'a:contains("Word")': 1
     });
   });
+
+  it('TINY-8057: Checking that mceLink command can open Quicklink and dialog', async () => {
+    const editor = hook.editor();
+    editor.setContent('');
+    editor.execCommand('mcelink', false, true);
+    await TinyUiActions.pWaitForDialog(editor);
+    TinyUiActions.closeDialog(editor);
+    editor.execCommand('mcelink');
+    // Waiting for Quicklink dialog as in pOpenQuickLink
+    await Waiter.pWait(100);
+    await FocusTools.pTryOnSelector('Selector should be in context form input', doc, '.tox-toolbar input');
+  });
+
+  it('TINY-8057: Checking Quicklink opens with keyboard shortcut', async () => {
+    const editor = hook.editor();
+    editor.setContent('<p>blah blah blah</p>');
+    TinyUiActions.keystroke(editor, 0x001E, metaKey); // Meta+K
+    TinyUiActions.keystroke(editor, 75, metaKey); // Meta+K
```

Comment:
```suggestion
    TinyUiActions.keystroke(editor, 'k'.charCodeAt(0), metaKey);
```
This makes it more readable and so that you don't need the comment.